### PR TITLE
Workflow Automated Test - LRQA-68836 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
Just started reviewing :)

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -387,12 +387,37 @@ definition {
...
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:401]: In this `.testcase` file, the `setUp` already assign Blogs to **single approver**, Please remove this piece. Thank you!

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by PauloVAS** [comment]
@lucasperj, fixed!

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**